docs/navigate.md

```diff
@@ -57,6 +60,21 @@ update itself while the user is panning around on it (set to null to disable
 intermediate pans, the plot will then not update until the mouse button is
 released).
 
+**mode** a string specifies which pan mode to be use. Accepted values:
+empty string for normal mode without pan hint or direction snapping;
+'smart' for smart mode with pan hint. The graph movement will snap to one direction
```

Comment:
Describing 'smart' with the word "smart" ("'smart' for **smart**...") isn't helpful. The second sentence should be all that is needed.

Reply:
Will be fixed in the following commits.